ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/OVERLAY_ROUTING_PROTOCOL_CVX_L3LEAF1.cfg

```diff
@@ -0,0 +1,72 @@
+!RANCID-CONTENT-TYPE: arista
+!
+vlan internal order ascending range 1006 1199
+!
+transceiver qsfp default-mode 4x10G
+!
+service routing protocols model multi-agent
+!
+hostname OVERLAY_ROUTING_PROTOCOL_CVX_L3LEAF1
+!
+no enable password
+no aaa root
+!
+vlan 2910
+   name L2VLAN_2910
+!
+vlan 2911
+   name L2VLAN_2911
+!
+vrf instance MGMT
+!
+interface Loopback0
+   description EVPN_Overlay_Peering
```

Comment:
Should this be CVX_Overlay_Peering?
Also, should we standardize this to all caps?

Reply:
We are not modifying descriptions. As we have talked about forever, this description is misleading anyway.

Reply:
Ok breaking change in 5.0 then :-)